sites/svelte.dev/src/lib/components/ReplWidget.svelte

```diff
@@ -58,15 +57,12 @@
 					repl.set({ components });
 				});
 		} else if (example) {
-			fetch(`${PUBLIC_API_BASE}/docs/svelte/examples/${example}`).then(async (response) => {
-				if (response.ok) {
-					const data = await response.json();
-					const components = process_example(data.files);
-
-					repl.set({
-						components,
-					});
-				}
+			const components = process_example(
+				getContext('repl_widget_examples').find(({ id }) => id === example).files
```

Comment:
`getContext` could be called outside of component initialization here, I suggest to move the context retrieval part out to the top level.

Reply:
Done!